### ISSUE TITLE
Can't find path in fbs freeze

### ISSUE THREAD

**Issue by yishenggudou** [mherrmann/fbs#146]
when i run `fbs freeze`

```
12707 ERROR: Can not find path ./QtMultimediaQuick.framework/Versions/5/QtMultimediaQuick (needed by /Users/timger/Documents/GitHub/android-controller/venv/lib/python3.6/site-packages/PyQt5/Qt/qml/QtMultimedia/libdeclarative_multimedia.dylib)
12849 ERROR: Can not find path ./Qt3DLogic.framework/Versions/5/Qt3DLogic (needed by /Users/timger/Documents/GitHub/android-controller/venv/lib/python3.6/site-packages/PyQt5/Qt/qml/QtQuick/Scene2D/libqtquickscene2dplugin.dylib)
12849 ERROR: Can not find path ./Qt3DInput.framework/Versions/5/Qt3DInput (needed by /Users/timger/Documents/GitHub/android-controller/venv/lib/python3.6/site-packages/PyQt5/Qt/qml/QtQuick/Scene2D/libqtquickscene2dplugin.dylib)
12849 ERROR: Can not find path ./Qt3DQuickScene2D.framework/Versions/5/Qt3DQuickScene2D (needed by /Users/timger/Documents/GitHub/android-controller/venv/lib/python3.6/site-packages/PyQt5/Qt/qml/QtQuick/Scene2D/libqtquickscene2dplugin.dylib)
12849 ERROR: Can not find path ./QtGamepad.framework/Versions/5/QtGamepad (needed by /Users/timger/Documents/GitHub/android-controller/venv/lib/python3.6/site-packages/PyQt5/Qt/qml/QtQuick/Scene2D/libqtquickscene2dplugin.dylib)
12850 ERROR: Can not find path ./Qt3DCore.framework/Versions/5/Qt3DCore (needed by /Users/timger/Documents/GitHub/android-controller/venv/lib/python3.6/site-packages/PyQt5/Qt/qml/QtQuick/Scene2D/libqtquickscene2dplugin.dylib)
12850 ERROR: Can not find path ./Qt3DRender.framework/Versions/5/Qt3DRender (needed by /Users/timger/Documents/GitHub/android-controller/venv/lib/python3.6/site-packages/PyQt5/Qt/qml/QtQuick/Scene2D/libqtquickscene2dplugin.dylib)
12851 ERROR: Can not find path ./Qt3DQuick.framework/Versions/5/Qt3DQuick (needed by /Users/timger/Documents/GitHub/android-controller/venv/lib/python3.6/site-packages/PyQt5/Qt/qml/QtQuick/Scene2D/libqtquickscene2dplugin.dylib)
12960 ERROR: Can not find path ./Qt3DLogic.framework/Versions/5/Qt3DLogic (needed by /Users/timger/Documents/GitHub/android-controller/venv/lib/python3.6/site-packages/PyQt5/Qt/qml/QtQuick/Scene3D/libqtquickscene3dplugin.dylib)
12961 ERROR: Can not find path ./Qt3DInput.framework/Versions/5/Qt3DInput (needed by /Users/timger/Documents/GitHub/android-controller/venv/lib/python3.6/site-packages/PyQt5/Qt/qml/QtQuick/Scene3D/libqtquickscene3dplugin.dylib)
12961 ERROR: Can not find path ./QtGamepad.framework/Versions/5/QtGamepad (needed by /Users/timger/Documents/GitHub/android-controller/venv/lib/python3.6/site-packages/PyQt5/Qt/qml/QtQuick/Scene3D/libqtquickscene3dplugin.dylib)
12961 ERROR: Can not find path ./Qt3DRender.framework/Versions/5/Qt3DRender (needed by /Users/timger/Documents/GitHub/android-controller/venv/lib/python3.6/site-packages/PyQt5/Qt/qml/QtQuick/Scene3D/libqtquickscene3dplugin.dylib)
12962 ERROR: Can not find path ./Qt3DCore.framework/Versions/5/Qt3DCore (needed by /Users/timger/Documents/GitHub/android-controller/venv/lib/python3.6/site-packages/PyQt5/Qt/qml/QtQuick/Scene3D/libqtquickscene3dplugin.dylib)
12962 ERROR: Can not find path ./Qt3DAnimation.framework/Versions/5/Qt3DAnimation (needed by /Users/timger/Documents/GitHub/android-controller/venv/lib/python3.6/site-packages/PyQt5/Qt/qml/QtQuick/Scene3D/libqtquickscene3dplugin.dylib)
Done. You can now run `target/acu.app/Contents/MacOS/acu`. If that

```
os info:
 `Darwin yangyi-mac-mini.local 18.0.0 Darwin Kernel Version 18.0.0: Wed Aug 22 20:13:40 PDT 2018; root:xnu-4903.201.2~1/RELEASE_X86_64 x86_64`

Python version:

```
(venv) ➜  android-controller git:(feature/python3-support) ✗ python --version
Python 3.6.8
```

fab verison

```
fbs==0.8.4
```

PyInstaller version
```
PyInstaller==3.4
```

PyQt5 version
```
PyQt5==5.9.2
```

Thanks!

**Comment by mherrmann** [comment]
Hi, do you have a minimal example that reproduces this problem?

**Comment by yishenggudou** [comment]
@mherrmann  

Thank you for your reply

no minimal example

I have change to use PySide2 

Add the following code to my `Makefile`

It can build a mac app successful

```
cp -vf venv/lib/python3.6/site-packages/shiboken2/libshiboken2.abi3.5.13.dylib venv/lib/python3.6/site-packages/PyInstaller/hooks/
cp -vf venv/lib/python3.6/site-packages/shiboken2/libshiboken2.abi3.5.13.dylib  venv/lib/python3.6/site-packages/PySide2/
fbs freeze
mkdir -p target/acu.app/Contents/MacOS/ui
cp -vrf src/main/python/ui/*  target/acu.app/Contents/MacOS/ui/
```

the ui directory include all my `qml` files

**Comment by asukakenji** [comment]
The above workaround by @yishenggudou is helpful!

Basically, I copied `libshiboken2.abi3.xx.yy.dylib` (where `xx` and `yy` are version numbers) from `site-packages/shiboken2/` to both `site-packages/PyInstaller/hooks/` and `site-packages/PySide2/`, then the build was executed successfully.

However, the "*.app" was still not built correctly. When it was executed, the error `LSOpenURLsWithRole() failed with error -10810` occurred. After some researches, I found this comment to be helpful:
https://stackoverflow.com/questions/50580039/creating-pyinstaller-from-my-script-missing-pyside2-qtxml-in-the-result#comment104850986_55771925

Basically, I added the following import to the main source file fix the problem:
```py
from PySide2 import QtXml
```
